package/MDAnalysis/analysis/dihedrals.py

```diff
@@ -101,13 +146,63 @@
 import MDAnalysis as mda
 from MDAnalysis.analysis.base import AnalysisBase
 from MDAnalysis.lib.distances import calc_dihedrals
+from MDAnalysisTests.datafiles import Rama_ref, Janin_ref
```

Comment:
We can't reference MDAnalysisTests in MDAnalysis because people might not have the tests installed.

Instead, for right now, create a directory ``analysis/data`` and deposit your files there.

You will also need to make sure that `package/setup.py` actually includes your data files when you do a
```
python setup.py sdist
```

Have a look at the `testsuite/setup.py` for how to include datafiles https://github.com/MDAnalysis/mdanalysis/blob/10fb62d95f52e7b6b77870a53a35e7adbc477658/testsuite/setup.py#L183 with the `package_data` keyword.